packages/vm/src/index.ts

```diff
@@ -276,10 +292,13 @@ export default class VM extends AsyncEventEmitter {
     }
     this._common.on('hardforkChanged', () => {
       this.getActiveOpcodes()
+      this._precompiles = getActivePrecompiles(this._common, this._customPrecompiles)
     })
 
-    // Initialize the opcode data
+    // Set list of opcodes based on HF
+    // TODO: make this EIP-friendly
```

Comment:
What does this mean?  Is this something that needs to be resolved before this can be merged?

Reply:
I accidentally copied this from `master` when I started this, it is also gone in there, so I removed it!